paz/models/classification/vvad_lrs3.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+
+keras = tf.keras
+from keras.models import Model
+from keras.layers import (Input, BatchNormalization, Flatten, Dense, LSTM, TimeDistributed)
+from keras.applications.mobilenet import MobileNet
+import random
+
+def VVAD_LRS3_LSTM(weights=None, input_shape=(38, 96, 96, 3), seed=305865, tmp_weights_path="../../../../CLUSTER_OUTPUTS/VVAD_LRS3/2023_10_30-09_51_57/vvad_lrs3-weights-57.hdf5"):
```

Comment:
I don't know if passing the seed here is a good idea. I would leave all randomization to a higher global variable.

Reply:
We decided to leave it in there, as it is part of my thesis. But in the future I will use keras.utils.set_random_seed.